x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/form/schema.ts

```diff
@@ -30,6 +30,90 @@ const serializers = {
   stringToNumber: (v: string): any => (v != null ? parseInt(v, 10) : undefined),
 };
 
+const maxNumSegmentsField = {
+  label: i18nTexts.editPolicy.maxNumSegmentsFieldLabel,
+  validations: [
+    {
+      validator: emptyField(
+        i18n.translate(
+          'xpack.indexLifecycleMgmt.editPolicy.forcemerge.numberOfSegmentsRequiredError',
+          { defaultMessage: 'A value for number of segments is required.' }
+        )
+      ),
+    },
+    {
+      validator: ifExistsNumberGreaterThanZero,
+    },
+  ],
+  serializer: serializers.stringToNumber,
+};
+
+export const searchableSnapshotFields = {
+  snapshot_repository: {
+    label: i18nTexts.editPolicy.searchableSnapshotsRepoFieldLabel,
+    validations: [
+      { validator: emptyField(i18nTexts.editPolicy.errors.searchableSnapshotRepoRequired) },
+    ],
+  },
+  storage: {
+    label: i18n.translate('xpack.indexLifecycleMgmt.editPolicy.searchableSnapshot.storageLabel', {
+      defaultMessage: 'Storage',
+    }),
+    helpText: i18n.translate(
+      'xpack.indexLifecycleMgmt.editPolicy.searchableSnapshot.storageHelpText',
+      {
+        defaultMessage:
+          'Specifies the type of snapshot that should be mounted for a searchable snapshot. Warning: this is an advanced settings, only change it if you know what you do.',
```

Comment:
Just made some minor tweaks for grammar here.
```suggestion
          "Type of snapshot mounted for the searchable snapshot. This is an advanced option. Only change it if you know what you're doing.",
```